internal/segment/segment.go

```diff
@@ -1,23 +1,25 @@
 package segment
 
 import (
+	"runtime"
+
 	"github.com/rs/zerolog/log"
 	"github.com/saucelabs/saucectl/internal/ci"
 	"github.com/saucelabs/saucectl/internal/credentials"
 	"github.com/saucelabs/saucectl/internal/setup"
 	"github.com/saucelabs/saucectl/internal/usage"
 	"github.com/saucelabs/saucectl/internal/version"
 	"gopkg.in/segmentio/analytics-go.v3"
-	"runtime"
 )
 
 // Tracker is the segment implemention for usage.Tracker.
 type Tracker struct {
-	client analytics.Client
+	client     analytics.Client
+	IsDisabled bool
```

Comment:
Reserve Is/Has etc. for methods when you can.

```suggestion
	Enabled bool
```